src/server/req_rescq.c

```diff
@@ -175,17 +175,16 @@ cnvrt_qmove(resc_resv  *presv)
 	reqcnvrt->rq_perm = (presv->ri_brp)->rq_perm;
 	strcpy(reqcnvrt->rq_user, (presv->ri_brp)->rq_user);
 	strcpy(reqcnvrt->rq_host, (presv->ri_brp)->rq_host);
-	len = strlen(q_job_id);
 
-	strncpy(reqcnvrt->rq_ind.rq_move.rq_jid, q_job_id, len);
-	reqcnvrt->rq_ind.rq_move.rq_jid[strlen(q_job_id)] = '\0';
+	strncpy(reqcnvrt->rq_ind.rq_move.rq_jid, q_job_id,
+		sizeof(reqcnvrt->rq_ind.rq_move.rq_jid) - 1);
+	reqcnvrt->rq_ind.rq_move.rq_jid[sizeof(reqcnvrt->rq_ind.rq_move.rq_jid) - 1] = '\0';
```

Comment:
Might want to just save the value of  'sizeof(reqcnvrt->rq_ind.rq_move.rq_jid) - 1' since you repeat it twice.

Reply:
Done.